docs/releasenotes/service.md

```diff
@@ -9,6 +9,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Service Release Notes for the current year are listed here. If you need to see previous Service Release Notes, check the [Archive](docs/releasenotes/archive).
 
+
+---
+## September 22, 2022 (Security)
+
+Collection
+Update - Good news! We've increased the number of Cloud-to-Cloud Sources that customers with paid Sumo Logic subscriptions can have, from 20 to 50. For users of free accounts, the limit remains 20 Cloud-to-Cloud Sources. For more information, see [Cloud-to-Cloud Integration Framework] (/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/).
```

Comment:
@stacykor  fix broken link

Reply:
Hi @stacykor , to merge this PR, you'll also need to click the "Resolve Conversation" button.